src/components/Grid.scss

```diff
@@ -62,6 +64,7 @@ $Grid-gutter: $su-medium;
 
 .Grid--withGutter {
   margin: 0 (-0.5 * $Grid-gutter);
+  width: calc(100% + 16px); // 2
```

Comment:
I'm cool with this approach.

Re: variable: I think this might work with interpolation: `#{$Grid-gutter}`... also I think Sass has operators prebaked, so you may not need calc at all, like in the above margin expression. http://sass-lang.com/guide#topic-8

Reply:
1. ah that's a good call on the #{$Grid-gutter}. Updated.

2. That's definitely an interesting approach! and works well for specific scenarios because we know the max width of a given element (full container for example is 1300px). So `width: ($max-width + $Grid-gutter) / $max-width  * 100%;` covers the overview screen but it doesn't seem to work for the checkout page because we don't know what that container width is in terms of a pixel value, and may not know what it is in future situations.  In the example too they're working off set values to create those percentages, just don't _think_ we have that luxury here. Whereas calc(100%) always returns what that container width ends up being in the browser and let's us add on to it.

Reply:
Cool 👍 Yeah, I was thinking of just using 100% + $Grid-gutter, but I didn't realize the operators don't function quite like `calc`, in that they _pre_ calculate versus calculate on render.